src/components/carousel/_index.scss

```diff
@@ -0,0 +1,152 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				opacity: 1;
+			}
+		}
+
+		&:focus {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				opacity: 1;
+			}
+		}
+
+		@include scss.component-properties("carousel-buttons");
+	}
+
+	.slick-next {
+		@include scss.component-properties("carousel-button-next");
+
+		&:before {
+			font-family: "Arial", serif;
+			content: ">";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+
+			@include scss.component-properties("carousel-button-next-before");
+		}
+	}
+
+	.slick-prev {
+		@include scss.component-properties("carousel-button-previous");
+
+		&:before {
+			font-family: "Arial", serif;
+			content: "<";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+
+			@include scss.component-properties("carousel-button-previous-before");
+		}
+	}
+
+	.slick-prev.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+	}
+
+	.slick-next.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+	}
+
+	[dir="rtl"] {
+		.slick-prev {
+			@include scss.component-properties("carousel-button-previous");
+			left: auto;
+
+			&:before {
+				content: ">";
+			}
+		}
+
+		.slick-next {
+			right: auto;
+			@include scss.component-properties("carousel-button-next");
+
+			&:before {
+				content: "<";
+			}
+		}
+	}
+
+	/* Dots */
+	.slick-dotted.slick-slider {
+		@include scss.component-properties("carousel-slider-bottom");
```

Comment:
The name "bottom" might be too positionally specific.  What if someone puts their navigation dots on top?